components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -32,30 +34,44 @@ const DEFAULT_MAX_SAMPLE_LOOP_COUNT: usize = 10000;
 pub const TOP_N: usize = 10;
 
 // LOAD_BASE_SPLIT_EVENT metrics label definitions.
-// Workload fits the QPS threshold or byte threshold.
-const LOAD_FIT: &str = "load_fit";
-// Workload fits the CPU threshold.
-const CPU_LOAD_FIT: &str = "cpu_load_fit";
-// The statistical key is empty.
-const EMPTY_STATISTICAL_KEY: &str = "empty_statistical_key";
-// Split info has been collected, ready to split.
-const READY_TO_SPLIT: &str = "ready_to_split";
-// Split info has not been collected yet, not ready to split.
-const NOT_READY_TO_SPLIT: &str = "not_ready_to_split";
-// The number of sampled keys does not meet the threshold.
-const NO_ENOUGH_SAMPLED_KEY: &str = "no_enough_sampled_key";
-// The number of sampled keys located on left and right does not meet the threshold.
-const NO_ENOUGH_LR_KEY: &str = "no_enough_lr_key";
-// The number of balanced keys does not meet the score.
-const NO_BALANCE_KEY: &str = "no_balance_key";
-// The number of contained keys does not meet the score.
-const NO_UNCROSS_KEY: &str = "no_uncross_key";
-// Split info for the top hot CPU region has been collected, ready to split.
-const READY_TO_SPLIT_CPU_TOP: &str = "ready_to_split_cpu_top";
-// Hottest key range for the top hot CPU region could not be found.
-const EMPTY_HOTTEST_KEY_RANGE: &str = "empty_hottest_key_range";
-// The top hot CPU region could not be split.
-const UNABLE_TO_SPLIT_CPU_TOP: &str = "unable_to_split_cpu_top";
+lazy_static! {
```

Comment:
please check spots calling `make_static_metric` for reference

Reply:
Changed. PTAL.